openwisp_notifications/templates/admin/base_site.html

```diff
@@ -37,11 +37,7 @@
       {% else %}
         const notificationApiHost = window.location;
       {% endif %}
-      {% if debug %}
-        const webSocketProtocol = 'ws';
-      {% else %}
-        const webSocketProtocol = 'wss';
-      {% endif %}
+      const webSocketProtocol = notificationApiHost.protocol === 'http' ? 'ws' : 'wss';
```

Comment:
I think it should be `'http:'`
On my end, the `'http'` match is failing.
Is it `http` only on your end?

Reply:
I think it was working using just ```http``` on my system. But I will try to check it again, could you please send your logs and how to reproduce it?